nservicebus/learning-transport/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Learning Transport
+component: LearningTransport
+reviewed: 2017-05-01
+---
+
+The Learning Transport stores all message actions in the local file system relative to the current solution directory.
+
+include: learning-transport-warning
+
+Added in Version 6.3.
+
+Some recommended use cases are:
+
+ * Learning/Experimenting with NServiceBus features.
+ * Building a [spike](https://en.wikipedia.org/wiki/Spike_(software_development)) or [demo](https://simple.wikipedia.org/wiki/Demo_(software)).
+ * Reproducing a bug that is not related to a production transport, for example when raising a support case.
+
+Currently [ServiceControl](/servicecontrol/) (and hence [ServicePulse](/servicepulse/) and [ServiceInsight](https://docs.particular.net/serviceinsight/)) are not supported.
+
+
+## Usage
+
+snippet: LearningTransport
+
+
+## File System Structure
```

Comment:
Out of interest, can I lock these files if I open them in a text editor while the system is running? What happens?